### DISCUSSION TITLE
Cloning the repo Error!

### DISCUSSION THREAD

**Discussion by jornejongsma** [Via-Appia/via-appia-online-viewer#37] [General]
I tried to clone the repo this morning and received this error.
Anyone familiar with this?
```
Cloning into 'via-appia-online-viewer'...
remote: Enumerating objects: 2021, done.
remote: Counting objects: 100% (154/154), done.
remote: Compressing objects: 100% (94/94), done.
remote: Total 2021 (delta 76), reused 113 (delta 56), pack-reused 1867
Receiving objects: 100% (2021/2021), 50.54 MiB | 5.54 MiB/s, done.
Resolving deltas: 100% (444/444), done.
error: invalid path 'static/images/pyramid.jpg:Zone.Identifier'
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
```

**Comment by jornejongsma** [comment]
Well, it indeed seems to be Git that is the trouble maker.... On another computer I had an older version of Git running, and that seems to work fine, so I re-installed the older version on this one and it works! (Git 2.30.windows.2 is NOT working) (Git 2.23.windows.1 works)

Case closed..!

**Reply by ctwhome** [reply]
I need to add more documentation on how to run the application, and what is git ignored.
